src/components/BusinessCaseReview/index.tsx

```diff
@@ -83,7 +83,7 @@ const BusinessCaseReview = ({
             Alternatives analysis
           </h2>
         </div>
-        <div className="alternative-analysis-wrapper">
+        <div className="padding-bottom-8 alternative-analysis-wrapper">
```

Comment:
I think these should be undone - looks like it might be from @aterstriep's changes in https://github.com/CMSgov/easi-app/pull/1667